### PR TITLE
Translating ch07-01 in French.

### LINKED ISSUE #62: Translating ch07-01
I'm working on the French translation of this page.

### PR DESCRIPTION
**Pull request by Jimskapt**
Closes #62.

### CODE REVIEW

File: FRENCH/src/ch07-01-packages-and-crates.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+`Rng`. We can also define a `struct` named `Rng` in our own crate. Because a
+crate’s functionality is namespaced in its own scope, when we add `rand` as a
+dependency, the compiler isn’t confused about what the name `Rng` refers to. In
+our crate, it refers to the `struct Rng` that we defined. We would access the
+`Rng` trait from the `rand` crate as `rand::Rng`.
+-->
+
+Garder une fonctionnalité d'une crate dans sa propre portée clarifie si une
+fonctionnalité précise est définie dans notre crate ou dans la crate `rand` et
+évite ainsi des potentiels conflits. Par exemple, la crate `rand` fournit un
```
Comment (LeSeulArtichaut) [inline FRENCH/src/ch07-01-packages-and-crates.md]: Could probably be simplified to be easier to understand.

File: FRENCH/src/ch07-01-packages-and-crates.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+modules in depth in the [“Defining Modules to Control Scope and
+Privacy”][modules]<!-- ignore -- >) section. A *package* is one or more crates
+that provide a set of functionality. A package contains a *Cargo.toml* file
+that describes how to build those crates.
+-->
+
+La première partie du système de modules que nous allons aborder concerne les
+paquets et les *crates*. Une crate est un binaire ou une bibliothèque. La racine
+de la *crate* est un fichier source à partir duquel le compilateur Rust commence
+et crée le module racine de votre *crate* (nous verrons les modules plus en
```
[the remark this answers is not in the document]
Comment (Jimskapt) [reply FRENCH/src/ch07-01-packages-and-crates.md]: I agree, it looks like better that way ! 👍

File: FRENCH/src/ch07-01-packages-and-crates.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+<!--
+Several rules determine what a package can contain. A package *must* contain
+zero or one library crates, and no more. It can contain as many binary crates
+as you’d like, but it must contain at least one crate (either library or
+binary).
+-->
+
+Il y a plusieurs règles qui déterminent ce qu'un paquet peut contenir. Un paquet
+*doit* contenir entre zéro et une crate de bibliothèque, et pas plus. Il peut
```
[the remark this answers is not in the document]
Comment (Jimskapt) [reply FRENCH/src/ch07-01-packages-and-crates.md]: OK, but I think its missing the case of we have no library (although it seems very unlikely)

[the remark this answers is not in the document]
Comment (LeSeulArtichaut) [reply FRENCH/src/ch07-01-packages-and-crates.md]: I think the next sentence clarifies it enough
> il doit contenir
au moins une crate (que ce soit une bibliothèque ou un binaire).

[the remark this answers is not in the document]
Comment (Jimskapt) [reply FRENCH/src/ch07-01-packages-and-crates.md]: Attempting to correct in 6cc61cb6b9cac5688f9243fec4ab5383b00b154c

File: FRENCH/src/ch07-01-packages-and-crates.md (hunk 1 above)
Comment (Jimskapt) [reply FRENCH/src/ch07-01-packages-and-crates.md]: Attempting to correct in 6cc61cb6b9cac5688f9243fec4ab5383b00b154c

**Pull request merged**